addons/sms/models/sms_sms.py

```diff
@@ -119,7 +119,7 @@ def resend_failed(self):
             if success_sms > 0:
                 notification_title = _('Success')
                 notification_type = 'success'
-                notification_message = _('%s out of the %s selected SMS Text Messages have successfully been resent.', success_sms, len(self))
+                notification_message = _('%(count)s out of the %(total)s selected SMS Text Messages have successfully been resent.', count=success_sms, total=len(self))
```

Comment:
While we're at it, this lacks various plural forms, but I guess it's not the scope of this PR.